src/main/scala/eventstore/tcp/ConnectionActor.scala

```diff
@@ -277,7 +279,7 @@ private[eventstore] class ConnectionActor(settings: Settings) extends Actor with
     case Connected(`address`) =>
       log.info("Connected to {}", address)
       val connection = Connection(address, sender(), context)
-
+      connection(PackOut(identifyClient))
```

Comment:
@t3hnar Is this the only thing required?

Reply:
Yep, looks so